readme.md

```diff
@@ -1,38 +1,86 @@
 Bullet Utils
-----------------
-
-### What it is
+------
 
 The code provided in this repo simplifies the interaction between pinocchio and pybullet.
 
-### Get started
+### Installation
 
-If you already have Pinocchio, the installation is simple:
-```
-git clone git@github.com:machines-in-motion/bullet_utils.git
-cd bullet_utils
-pip3 install .
-```
+#### Standard dependencies
+
+You need [PyBullet](https://github.com/bulletphysics/bullet3) and
+[Pinocchio](https://github.com/stack-of-tasks/pinocchio) to install this
+package.
+
+To run the demo, you need to install
+[robot_properties_solo](https://github.com/open-dynamic-robot-initiative/robot_properties_solo)
+as well.
+
+#### Download the package
+
+To download this package, you can
+
+1. use `git clone`  
+    ```
+    mkdir -p ~/devel/workspace/src
+    cd ~/devel/workspace/src
+    git clone git@github.com:machines-in-motion/bullet_utils.git
+    ```
+
+2. use [treep](https://gitlab.is.tue.mpg.de/amd-clmc/treep) with the [treep_machines_in_motion](https://github.com/machines-in-motion/treep_machines_in_motion) configuration.  
+    ```
+    mkdir -p ~/devel
+    pip3 install treep
+    cd ~/devel
+    git clone git@github.com:machines-in-motion/treep_machines_in_motion.git
+    treep --clone bullet_utils
+    ```
 
-To run the demo, you need to install robot_properties_solo
+#### Build the package
+
+You can install this package with 
+- pure pip  
+    ```
+    cd mkdir -p ~/devel/workspace/src/bullet_utils
+    pip3 install .
+    ```
+
+- or [colcon](https://github.com/machines-in-motion/machines-in-motion.github.io/wiki/use_colcon)  
+    ```
+    cd mkdir -p ~/devel/workspace
+    colcon build
+    ```
+
+### Usage
+
+#### Demos
+
+To run the demo
 ```
-git clone git@github.com:open-dynamic-robot-initiative/robot_properties_solo.git
-cd robot_properties_solo
-pip3 install .
+cd mkdir -p ~/devel/workspace/src/bullet_utils/demos
+python3 demo_simulate_a_robot.py
 ```
 
-### Authors
+#### API documentation
 
-- Julian Viereck
-- Maximilien Naveau
-- Huaijiang Zhu
+The API documentation is automatically generated using colcon.
+For the build to work one need to add the
+[mpi_cmake_modules](https://github.com/machines-in-motion/mpi_cmake_modules)
+to their workspace using git or treep.
 
-### Copyrights
+```
+cd ~/devel/workspace/src
+git clone git@github.com:machines-in-motion/mpi_cmake_modules.git
+```
+or
+```
+treep --clone mpi_cmake_modules
+```
 
-Copyright(c) 2018-2021 Max Planck Gesellschaft, New York University
+TODO: *Where to find the last built doc on the internet.*
```

Comment:
Do you want to keep this TODO in the official readme?

Reply:
yes because I want to remember to put a link here

Reply:
we can comment this out but I don't know how to...

Reply:
I don't think there is a comment feature for Markdown. So you would have to remove it.